defender/downloaded/defender-config.json

```diff
@@ -0,0 +1,326 @@
+{
+  "sentinels": [
+    {
+      "name": "Compound Large Borrows Governance",
+      "paused": false,
+      "type": "FORTA",
+      "fortaConditions": {
+        "severity": 1,
+        "alertIDs": [
+          "AE-COMP-GOVERNANCE-THRESHOLD"
+        ],
+        "minimumScannerCount": 1
+      },
+      "addresses": [],
+      "agentIDs": [
+        "0x704a35d97dfae2caf3b409206f1bc4c8a06671170108d07f9b3c7ea5026916fc"
+      ],
+      "alertTimeoutMs": 0,
+      "notificationChannels": [
+        "Philip Compound"
+      ]
+    },
+    {
+      "type": "BLOCK",
+      "network": "mainnet",
+      "confirmLevel": 1,
+      "paused": false,
+      "name": "cToken Monitor",
+      "addresses": [
+        "0xe65cdB6479BaC1e22340E4E755fAE7E509EcD06c",
+        "0x6C8c6b02E7b2BE14d4fA6022Dfd6d75921D90E4E",
+        "0x70e36f6BF80a52b3B46b3aF8e106CC0ed743E8e4",
+        "0x5d3a536E4D6DbD6114cc1Ead35777bAB948E3643",
+        "0x7713DD9Ca933848F6819F38B8352D9A15EA73F67",
+        "0xFAce851a4921ce59e912d19329929CE6da6EB0c7",
+        "0x95b4eF2869eBD94BEb4eEE400a99824BF5DC325b",
+        "0x158079Ee67Fce2f58472A96584A73C7Ab9AC95c1",
+        "0xF5DCe57282A584D2746FaF1593d3121Fcac444dC",
+        "0x4B0181102A0112A2ef11AbEE5563bb4a3176c9d7",
+        "0x12392F67bdf24faE0AF363c24aC620a2f67DAd86",
+        "0x35A18000230DA775CAc24873d00Ff85BccdeD550",
+        "0x39AA39c021dfbaE8faC545936693aC917d5E7563",
+        "0x041171993284df560249B57358F931D9eB7b925D",
+        "0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9",
+        "0xC11b1268C1A384e55C48c2391d8d480264A3A7F4",
+        "0xccF4429DB6322D5C611ee964527D42E5d685DD6a",
+        "0x80a2AE356fc9ef4305676f7a3E2Ed04e12C33946",
+        "0xB3319f5D18Bc0D84dD1b4825Dcde5d5f7266d407"
+      ],
+      "abi": "./downloaded/cToken_Monitor_ABI.json",
```

Comment:
for this abi path (as well as the few others below), would it just be `./cToken_Monitor_ABI.json` because our current `defender-config.json` file is already in the downloaded folder and not in the main folder

Reply:
unless it looks like it's pulling down from the home path of the directory and in that case it would be correct as it is?

Reply:
The context for loading this file is the main directory, so the `./downloaded` path prefix is necessary.